src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.BoxedTypes.cs

```diff
@@ -548,7 +548,9 @@ public override MethodIL EmitIL()
 
                 // unbox to get a pointer to the value type
                 codeStream.EmitLdArg(0);
-                codeStream.Emit(ILOpcode.ldflda, emit.NewToken(boxedValueField));
+                MetadataType helperType = Context.SystemModule.GetKnownType("System", "Object");
```

Comment:
`ldfda` on `GetKnownType("System.Runtime.CompilerServices", "RawData").GetField("Data")` would work too and it would be a bit more efficient (no call).

Reply:
I can move `RawData` out of `Object.CoreRT.cs` into `System.Runtime.CompilerServices` is that right?